PowerShell/JumpCloud Module/Private/NestedFunctions/Get-JCResults.ps1

```diff
@@ -5,7 +5,8 @@ Function Get-JCResults
         [Parameter(Mandatory = $true, HelpMessage = 'URL of Endpoint')][ValidateNotNullOrEmpty()]$URL,
         [Parameter(Mandatory = $true, HelpMessage = 'Method of WebRequest')][ValidateNotNullOrEmpty()]$method,
         [Parameter(Mandatory = $true, HelpMessage = 'Limit of WebRequest')][ValidateNotNullOrEmpty()]$limit,
-        [Parameter(Mandatory = $false, HelpMessage = 'Body of WebRequest, if required')]$body
+        [Parameter(Mandatory = $false, HelpMessage = 'Body of WebRequest, if required')]$body,
+        [Parameter(Mandatory = $false, HelpMessage = 'Boolean: True to run in parallel, False to run in sequential; Default value: false')][bool]$parallel = $false
```

Comment:
Great, I think after we are done and confident we can switch this to default to parallel